docker/docker_client.go

```diff
@@ -165,8 +165,13 @@ func dockerCertDir(sys *types.SystemContext, hostPort string) (string, error) {
 	var (
 		hostCertDir     string
 		fullCertDirPath string
+		homeCertDir     string
 	)
-	for _, systemPerHostCertDirPath := range systemPerHostCertDirPaths {
+	home := os.Getenv("HOME")
+	if home != "" {
+		homeCertDir = filepath.Join(home, ".config", "containers", ".certs.d")
```

Comment:
Please make this a global constant defined near `systemPerHostCertDirPaths` (compare `dockerHomePath`).

Reply:
Well made it a global var, which matches dockerHomePath